src/js/pages/nodes/NodeDetailPage.js

```diff
@@ -0,0 +1,373 @@
+
+    let maxY = 5;
+    totalResources[resource].forEach(function (resourceTotal) {
+      if (resourceTotal.percentage > maxY) {
+        maxY = resourceTotal.percentage;
+      }
+    });
+
+    maxY *= 1.5; // Multiply by 150%
+    maxY /= 10; // Divide so that we can round to nearest tenth
+    maxY = Math.round(maxY); // Round
+    maxY *= 10; // Multiply so that we get a percentage number between 0-100
+    maxY = Math.min(100, maxY); // Don't let it be greater than 100%
+
+    return (
+      <div key={resource} className="column-12
+        flex-box-align-vertical-center
+        container-pod
+        container-pod-super-short
+        flush-top">
+        <div className="media-object-spacing-wrapper media-object-offset">
+          <div className="media-object media-object-align-middle">
+            <div className="media-object-item">
+              <i className={resourceIconClasses}></i>
+            </div>
+            <div className="media-object-item">
+              <h4 className="flush-top flush-bottom inverse">
+                {resourceValue}
+              </h4>
+              <span className={`side-panel-resource-label
+                  text-color-${colorIndex}`}>
+                {resourceLabel.toUpperCase()}
+              </span>
+            </div>
+          </div>
+        </div>
+
+        <Chart calcHeight={function (w) { return w / WIDTH_HEIGHT_RATIO; }}
+          delayRender={true}>
+          <BarChart
+            axisConfiguration={axisConfiguration}
+            data={resourceData}
+            inverseStyle={true}
+            margin={{top: 0, left: 43, right: 10, bottom: 40}}
+            maxY={maxY}
+            refreshRate={Config.getRefreshRate()}
+            ticksY={3}
+            xGridLines={0}
+            y="percentage" />
+        </Chart>
+      </div>
+    );
+  }
+
+  getSubHeader(node) {
+    let activeTasksCount = node.sumTaskTypesByState('active');
+    let activeTasksSubHeader = StringUtil.pluralize('Task', activeTasksCount);
+    let healthStatus = node.getHealth();
+
+    return (
+      <ul className="list-inline flush-bottom">
+        <li>
+          <span className={healthStatus.classNames}>
+            {healthStatus.title}
+          </span>
+        </li>
+        <li>
+          {`${activeTasksCount} Active ${activeTasksSubHeader}`}
+        </li>
+      </ul>
+    );
+  }
+
+  renderHealthTabView() {
+    let node = this.internalStorage_get().node;
+    let units = NodeHealthStore.getUnits(node.hostname);
+
+    return (
+      <div className="container container-fluid flush">
+        <HealthTab
+          node={node}
+          units={units}
+          parentRouter={this.context.router} />
+      </div>
+    );
+  }
+
+  renderTasksTabView() {
+    let nodeID = this.props.params.nodeID;
+    let tasks = MesosStateStore.getTasksFromNodeID(this.props.params.nodeID);
+
+    let contents = this.getLoadingScreen();
+
+    let timeSinceMount = (Date.now() - this.mountedAt) / 1000;
```

Comment:
we can remove all of the `timeSinceMount` and `this.mountedAt` logic. that was there to not break animations. there are no more animations